Lib/ctypes/test/test_python_api.py

```diff
@@ -10,10 +10,8 @@
 ################################################################
 
 from sys import getrefcount as grc
-if sys.version_info > (2, 4):
-    c_py_ssize_t = c_size_t
-else:
-    c_py_ssize_t = c_int
+
+c_py_ssize_t = c_size_t
```

Comment:
Can `c_py_ssize_t` be removed altogether after it stopped to be a compatibility shim?

Reply:
Good idea! 👍